azure-pipelines-data-container.yml

```diff
@@ -0,0 +1,34 @@
+trigger:
+  - master
+  - datacontainerpipelinereview
```

Comment:
Include RPaaSMaster?

Reply:
that is in private repo. only master branch in public repo. need to clean up to remove datacontainerpipeline* branch before merging